tracing-scm-1.rockspec

```diff
@@ -37,6 +37,7 @@ build = {
 			['zipkin.reporter'] = 'tracing/zipkin/reporter.lua',
 			['zipkin.tracer'] = 'tracing/zipkin/tracer.lua',
 			['zipkin.bounded_queue'] = 'tracing/zipkin/bounded_queue.lua',
+			['tracing.version'] = 'tracing/version.lua',
```

Comment:
It is inconvenient to have a different `tracing` name. For example, when doing `package.loaded` cleanup, it is expected to clean all `module` and `module.*` packages. I think we should have separate `opentracing.version` and `zipkin.version` here. AFAIU, you can't specify the same file twice with the different install path, so it would result in two different version files (please, check it independently).

Reply:
Maybe I didn't quite understand, but now I have made separate `opentracing.version` and `zipkin.version` with the same `tracing/version.lua`. So, now `require` working with different args inside `tracing /opentracing/init.lua` and `tracing/zipkin/tracer.lua`

Reply:
This approach is fine too. (To be honest, I'm surprised that it is ok to use the same file in several luarocks targets -- I had failed to work this out in another PR.)

Reply:
It seems that it works for make, but doesn't work for builtin.